src/main/java/com/openshift/restclient/model/IResourceQuota.java

```diff
@@ -16,5 +16,29 @@
  * @author jeff.cantrill
  */
 public interface IResourceQuota extends IResource {
+	
+	public String getCpu();
+	
+	public void setCpu(String cpu);
+	
+	public String getMemory();
+	
+	public void setMemory(String memory);
+	
+	public String getPods();
```

Comment:
Should these return IPods, and same with the others IService, and IReplicatinControllers?